### PR TITLE
feat: [PL-32381]: CI widget on new account overview page

### PR DESCRIPTION
**Pull request by avinashmadhwani02**
#### Screenshots

Screenshot 2023-04-06 at 1 19 49 PM

Screenshot 2023-04-06 at 1 19 55 PM

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by abhinavrastogi-harness — DISMISSED** [review]
adding tests for this?

File: src/modules/45-projects-orgs/pages/LandingDashboardPageV2/ModuleOverview/modules/CIModuleOverview.tsx [hunk 1]
```diff
@@ -5,28 +5,111 @@
...
+        failureData.push(defaultTo(build.builds?.failed, 0))
+        custom.push(build)
+      })
+    }
+    const successCount = successData.reduce((sum, i) => sum + i, 0)
+    const failureCount = failureData.reduce((sum, i) => sum + i, 0)
+    const successArr = {
+      name: `Success (${successCount})`,
+      data: successData,
+      color: '#5FB34E',
```
Comment (mayankVermaHarness) [inline src/modules/45-projects-orgs/pages/LandingDashboardPageV2/ModuleOverview/modules/CIModuleOverview.tsx:56]: Should we use these colors from Design System?

File: src/modules/45-projects-orgs/pages/LandingDashboardPageV2/ModuleOverview/modules/CIModuleOverview.tsx [hunk 2]
```diff
@@ -5,28 +5,111 @@
...
+      custom
+    }
+    const failureArr = {
+      name: `Failed (${failureCount})`,
+      data: failureData,
+      color: '#EE5F54',
```
Comment (mayankVermaHarness) [inline src/modules/45-projects-orgs/pages/LandingDashboardPageV2/ModuleOverview/modules/CIModuleOverview.tsx:62]: 2nd: Should we use these colors from Design System?

**Review by mayankVermaHarness — APPROVED** [review]

File: src/modules/45-projects-orgs/pages/LandingDashboardPageV2/ModuleOverview/modules/CIModuleOverview.tsx [hunk 3]
```diff
@@ -5,28 +5,111 @@
...
+      buildInfoList.forEach(build => {
+        successData.push(defaultTo(build.builds?.success, 0))
+        failureData.push(defaultTo(build.builds?.failed, 0))
+        custom.push(build)
+      })
+    }
+    const successCount = successData.reduce((sum, i) => sum + i, 0)
+    const failureCount = failureData.reduce((sum, i) => sum + i, 0)
+    const successArr = {
+      name: `Success (${successCount})`,
```
Comment (mayankVermaHarness) [inline src/modules/45-projects-orgs/pages/LandingDashboardPageV2/ModuleOverview/modules/CIModuleOverview.tsx:54]: Should these strings come from string framework?

Comment (avinashmadhwani02) [reply src/modules/45-projects-orgs/pages/LandingDashboardPageV2/ModuleOverview/modules/CIModuleOverview.tsx]: Done

**Review by mayankVermaHarness — APPROVED** [review]

**Review by kapilkumawat86 — APPROVED** [review]

**Pull request merged**